README.md

```diff
@@ -13,39 +13,32 @@ The latest version is: **0.1**
 
   Requirements:
 	* Full latex distribution (for example miktex or texlive)
-	* For local SVG support: Installation of Poppler for Windows and Path setup of pdf2svg2.bat  https://community.jalios.com/jcms/jc2_183627/en/pdf2svg2-bat-script
-	* For local PNG support: Ghostscript
-		* The directory containing gswin32c.exe shall be added to PATH.
-		* ImageMagick (including legacy scripts) - rename convert.exe to imgconvert.exe
 
   Install:
 	1. Download Latex-Renderer
-	2. Make sure all required programs are installed.
 	3. 1st time Setup: *mvn package -DskipTests*
 	4. To start Latex-Renderer run: *mvn spring-boot:run*
 
 ### API:
 Runs on Port 8082.
 
-http://localhost:8082/renderLatex  expects (String content, List<String> latexPackages, String output) as JSON
+It is possible to Request the Data with either a POST- or a GET-Request
+* POST: http://localhost:8082/renderLatex  expects (String content, List<String> latexPackages, String output) as JSON
```

Comment:
I think a short example request for each case would be helpful - for example, how to specify multiple packages (for anyone how is not familiar with springboot and what parameters are expected in this case), what params are required